corehq/apps/app_manager/models.py

```diff
@@ -2658,20 +2497,6 @@ def uses_usercase(self):
     def grid_display_style(self):
         return self.display_style == 'grid'
 
-    def update_app_case_meta(self, meta):
-        from corehq.apps.reports.formdetails.readable import CaseMetaException
-
-        for column in self.case_details.long.columns:
-            try:
-                meta.add_property_detail('long', self.case_type, self.unique_id, column)
-            except CaseMetaException:
```

Comment:
Without the `except` blocks, will users now see errors that were previously silent?

Reply:
(meant to leave this comment on commit d5f9f1e12a5632ae9cb748009a53d74e8044a6f1)

Reply:
good question, but no. That `CaseMetaException` is caught further up the call chain and would never reach here.